typescript/trans/desugaring/desugar.str

```diff
@@ -50,7 +50,8 @@ rules
   
   desugar: ModuleItemsWithExport(items, item, items3) -> <concat> [items, [item], items3]
   desugar: ModuleItemsWithImport(items, item, items3) -> <concat> [items, [item], items3]
-	
+  
+  desugar: MethodFunction(name, callSignature, body) -> Function(FunctionName(name), callSignature, body)
```

Comment:
As a matter of fact... `FunctionDeclaration(name, sig, body)` can then be desugared into a `const name: sig = body` with a small transformation of the signature and body